### ISSUE TITLE
production ready?

### ISSUE THREAD

**Issue by guykatz** [gitawego/cordova-screenshot#33]
Hi all;
is this production ready for ios6+ and android 2+ (i see its v0.1.0)?
thanks.

**Comment by Gowrilink** [comment]
Hi 
Screenshot appear in camera role or temp file.in my case screen short placed in temp folder i am not seen that images.How we over come.

**Comment by nikhilknoldus** [comment]
No, @guykatz  this plugin still not production ready :( its been 2+ years but no ionic member is here to resolve the same issues related to captchuring canavs, iframe & videos. :-1:
